spec/controllers/state_file/questions/eligibility_offboarding_controller_spec.rb

```diff
@@ -18,12 +18,12 @@
   end
 
   describe "#edit" do
-    let(:intake) { create :state_file_az_intake }
+    let(:intake) { create :state_file_ny_intake }
 
     context "with offboarded_from set in the session" do
       render_views
       let(:offboarded_from_path) do
-        StateFile::Questions::AzEligibilityResidenceController.to_path_helper
+        StateFile::Questions::NyEligibilityResidenceController.to_path_helper
```

Comment:
🤔 hmm, why did y'all change out the controller referenced in this test to NY as opposed to another state that we support?